README.md

```diff
@@ -31,6 +31,9 @@ $ sudo add-apt-repository ppa:personalrobotics/ppa
 $ sudo apt-get update
 $ sudo apt-get install libaikido-all-dev
 ```
+-->
+
+<!--- TODO: Test on OS X and re-add to Homebrew
```

Comment:
I just wanted to flag this TODO in case it got overlooked

Reply:
Thanks for the flag! It's captured as an issue #604 

But note that, since nobody is on OS X anymore, this is very, very low priority.